### ISSUE TITLE
Can't update variables with space in the variable name

### ISSUE THREAD

**Issue by jvanderhoof** [cyberark/conjur-cli#233]
When trying to set the value of a variable with a space in its name through the CLI, I am unable to set a value.

I used the `conjurdemos/conjur-intro` repo to launch a Conjur instance:

```
$ ./run
```
In a new tab I loaded all the sample policy:
```
$ ./cli conjur policy load --replace root policy/users.yml
$ ./cli conjur policy load root policy/policy.yml
$ ./cli conjur policy load staging policy/apps/myapp.yml
$ ./cli conjur policy load production policy/apps/myapp.yml
$ ./cli conjur policy load root policy/application_grants.yml
$ ./cli conjur policy load root policy/hosts.yml
```

I tried the following:
```
./cli conjur variable values add "production/myapp/foo bar" foo-bar
./cli conjur variable values add production/myapp/foo%20;bar foo-bar
./cli conjur variable values add production/myapp/foo\ bar foo-bar
```
But was unable to set the value successfully.

**Comment by dataplex** [comment]
root@238565ae2f94:/policy# cat Issue_233.log
Script started on Tue Nov 13 21:37:19 2018
# curl -sk https://conjurmaster.cyberarkdemo.com/info
{
  "version": "5.2.3-rc1",
  "services": {
    "evoke": {
      "desired": "i",
      "status": "i",
      "err": null,
      "name": "conjur-evoke",
      "version": "5.12.0.2-9b8c78d",
      "arch": "amd64"
    },
    "ldap-sync": {
      "desired": "i",
      "status": "i",
      "err": null,
      "name": "conjur-ldap-sync",
      "version": "2.0.0.0-7303b0d",
      "arch": "amd64"
    },
    "possum": {
      "desired": "i",
      "status": "i",
      "err": null,
      "name": "conjur-possum",
      "version": "1.3.1.13-84abb49",
      "arch": "amd64"
    },
    "ui": {
      "desired": "i",
      "status": "i",
      "err": null,
      "name": "conjur-ui",
      "version": "2.10.10.0-70ecb8f",
      "arch": "amd64"
    },
    "ui-backend": {
      "desired": "i",
      "status": "i",
      "err": null,
      "name": "conjur-ui-backend",
      "version": "1.11.0.1-554c70e",
      "arch": "amd64"
    }
  },
  "role": "master",
  "configuration": {
    "conjur": {
      "role": "master",
      "account": "Conjurlab",
      "hostname": "cmaster1.cyberarkdemo.com",
      "cluster_name": "conjur",
      "cluster_master": "cmaster1.cyberarkdemo.com",
      "cluster_members": "cmaster1.cyberarkdemo.com=http://cmaster1.cyberarkdemo.com:2380,cmaster2.cyberarkdemo.com=http://cmaster2.cyberarkdemo.com:2380,cmaster3.cyberarkdemo.com=http://cmaster3.cyberarkdemo.com:2380",
      "cluster_machine_name": "cmaster1.cyberarkdemo.com",
      "cluster_machine_address": "cmaster1.cyberarkdemo.com"
    }
  },
  "authenticators": {
    "installed": [
      "authn",
      "authn-iam",
      "authn-k8s",
      "authn-ldap"
    ],
    "configured": [
      "authn"
    ],
    "enabled": [
      "authn"
    ]
  }
}# curl -sk https://conjurmaster.cyberarkdemo.com/health
{
  "services": {
    "possum": "ok",
    "ui-backend": "ok",
    "ui": "ok",
    "ok": true
  },
  "database": {
    "ok": true,
    "connect": {
      "main": "ok"
    },
    "free_space": {
      "main": {
        "kbytes": 22447520,
        "inodes": 13906177
      }
    },
    "replication_status": {
      "pg_stat_replication": [
        {
          "usename": "conjurmaster.cyberarkdemo.com",
          "application_name": "standby",
          "client_addr": "172.16.10.72",
          "backend_start": "2018-11-13 21:34:58 +0000",
          "state": "streaming",
          "sent_location": "0/4086540",
          "replay_location": "0/4086540",
          "sync_priority": 1,
          "sync_state": "sync",
          "sent_location_bytes": 67659072,
          "replay_location_bytes": 67659072,
          "replication_lag_bytes": 0
        },
        {
          "usename": "conjurmaster.cyberarkdemo.com",
          "application_name": "standby",
          "client_addr": "172.16.10.75",
          "backend_start": "2018-11-13 21:34:59 +0000",
          "state": "streaming",
          "sent_location": "0/4086540",
          "replay_location": "0/4086540",
          "sync_priority": 1,
          "sync_state": "potential",
          "sent_location_bytes": 67659072,
          "replay_location_bytes": 67659072,
          "replication_lag_bytes": 0
        }
      ],
      "pg_current_xlog_location": "0/4086540",
      "pg_current_xlog_location_bytes": 67659072
    }
  },
  "cluster": {
    "ok": true,
    "status": "running",
    "message": null
  },
  "ok": true
}# cat << EOF > issue233_root.yml
> - !policy production
> EOF
# cat conjurinit.sh
#!/usr/bin/env bash
set -x
#conjur init -u https://conjur-dev.demo.local/api -a ConjurDev
conjur init -u https://conjurmaster.cyberarkdemo.com -a Conjurlab
conjur authn login -u admin
# ./conjurinit.sh
+ conjur init -u https://conjurmaster.cyberarkdemo.com -a Conjurlab

SHA1 Fingerprint=5E:73:1B:D9:4A:5A:4C:5D:86:CE:C9:1A:DB:90:74:81:A0:E4:B0:8A

Please verify this certificate on the appliance using command:
              openssl x509 -fingerprint -noout -in ~conjur/etc/ssl/conjur.pem

Trust this certificate (yes/no): yes
Wrote certificate to /root/conjur-Conjurlab.pem
Wrote configuration to /root/.conjurrc
+ conjur authn login -u admin
Please enter admin's password (it will not be echoed):
Logged in
# conjur policy load root issue233_root.yml
Loaded policy 'root'
{
  "created_roles": {
  },
  "version": 2
}
# cat varswithspaces.yml
- !variable "myapp/foo bar"
# conjur policy load production varswithspaces.yml
Loaded policy 'production'
{
  "created_roles": {
  },
  "version": 1
}
# conjur variable values add "production/myapp/foo bar" SECRET1
Value added
# conjur variable value "production/myapp/foo bar"
SECRET1#
# conjur --version
conjur version 6.2.0
# conjur variable values add "production/myapp/foo bar" "foo bar 1"
Value added
# conjur variable value "production/myapp/foo bar"
foo bar 1# exit

Script done on Tue Nov 13 21:40:58 2018

**Comment by dataplex** [comment]
Resolved in 5.2.3-rc1.

# conjur variable values add "production/myapp/foo bar" SECRET1
Value added
# conjur variable value "production/myapp/foo bar"
SECRET1#
# conjur --version
conjur version 6.2.0
# conjur variable values add "production/myapp/foo bar" "foo bar 1"
Value added
# conjur variable value "production/myapp/foo bar"
foo bar 1# exit

**Comment by garkler** [comment]
Verified by @dataplex as solved with 5.2.3-rc1 with CLI 6.2.0
